portal/server/routes/contract/index.js

```diff
@@ -78,6 +81,7 @@ router.get('/contract/:_id', [provide([DEAL]), validateBank], async (req, res) =
     dealHasIssuedFacilitiesToSubmit: dealHasIssuedFacilitiesToSubmit(deal),
     confirmedRequestedCoverStartDates: confirmedRequestedCoverStartDates[dealId] || [],
     allRequestedCoverStartDatesConfirmed: deal.submissionType === 'Automatic Inclusion Notice' || allRequestedCoverStartDatesConfirmed,
```

Comment:
Appreciate this is not in your PR, but could you please update `Automatic Inclusion Notice` to be picked up from constant in `libs/common` please. Thanks.